packages/@lwc/engine-server/src/__tests__/fixtures.spec.ts

```diff
@@ -59,69 +58,12 @@ async function compileFixture({ input, dirname }: { input: string; dirname: stri
     return outputFile;
 }
 
-/**
- * Naive HTML fragment formatter.
- *
- * This is a replacement for Prettier HTML formatting. Prettier formatting is too aggressive for
- * fixture testing. It not only indent the HTML code but also fixes HTML issues. For testing we want
- * to make sure that the fixture file is as close as possible to what the engine produces.
- *
- * @param src the original HTML fragment.
- * @returns the formatter HTML fragment.
- */
 function formatHTML(src: string): string {
-    let res = '';
-    let pos = 0;
-    let start = pos;
-
-    let depth = 0;
-
-    const getPadding = () => {
-        return '  '.repeat(depth);
-    };
-
-    while (pos < src.length) {
-        // Consume element tags and comments.
-        if (src.charAt(pos) === '<') {
-            const tagNameMatch = src.slice(pos).match(/(\w+)/);
-
-            const isVoid = isVoidElement(tagNameMatch![0], HTML_NAMESPACE);
-            const isClosing = src.charAt(pos + 1) === '/';
-            const isComment =
-                src.charAt(pos + 1) === '!' &&
-                src.charAt(pos + 2) === '-' &&
-                src.charAt(pos + 3) === '-';
-
-            start = pos;
-            while (src.charAt(pos++) !== '>') {
-                // Keep advancing until consuming the closing tag.
-            }
-
-            // Adjust current depth and print the element tag or comment.
-            if (isClosing) {
-                depth--;
-            }
-
-            res += getPadding() + src.slice(start, pos) + '\n';
-
-            const isSelfClosing = src.charAt(pos - 2) === '/';
-            if (!isClosing && !isSelfClosing && !isVoid && !isComment) {
-                depth++;
-            }
-        }
-
-        // Consume text content.
-        start = pos;
-        while (src.charAt(pos) !== '<' && pos < src.length) {
-            pos++;
-        }
-
-        if (start !== pos) {
-            res += getPadding() + src.slice(start, pos) + '\n';
-        }
-    }
-
-    return res.trim();
+    return format(src, {
+        parser: 'html',
+        htmlWhitespaceSensitivity: 'ignore',
+        bracketSameLine: true,
+    });
```

Comment:
This is a nice idea, but I am not smart enough to maintain my own HTML parser/formatter. I could not figure out how to modify this to correctly handle unescaped characters like `>` and `<` inside of `<style>` tags.